### DISCUSSION TITLE
patchy lighting / fullbright

### DISCUSSION THREAD

**Discussion by TolleSache** [AnOpenSauceDev/Methane-mod#81] [Q&A]
sorry if im being stupid im new to methane (and not smart lmao) but when i have methane enabled it has the fullbright effect and no setting will change this, once methane is turned off i get this patchy effect of lighting during day and night. is there a way to fix this?

![2024-01-11_15 17 18](https://github.com/AnOpenSauceDev/Methane-mod/assets/156337918/ec65d016-9f5d-44fd-b4e2-2051b95ad953)
![2024-01-11_15 17 40](https://github.com/AnOpenSauceDev/Methane-mod/assets/156337918/8aca9228-d0a0-478c-a01c-7727225244f1)

**Comment by AnOpenSauceDev** [comment]
For some singleplayer worlds, (mainly those that haven't initially been made with vanilla) Methane will not correctly get the skylight of a chunk, and it results in these patches of light. There used to be a fix for this in very early versions of Methane, but it ended up breaking sodium IIRC. Creating a new world _might_ fix some of these issues.
